expo/scheduler/functions.py

```diff
@@ -435,15 +411,22 @@ def date_to_datetime(the_date):
     return utc.localize(datetime.combine(the_date, zero_hour))
 
 
+def send_today():
```

Comment:
Nice extraction, but I don't understand the name. This looks like it's constructing a time range for a full "day" from 8 AM  this morning to 4AM tonight/tomorrow morning. What is "send_today"?